sdk/e2e/11_deploy_router_upi_traffic_split_test.py

```diff
@@ -15,10 +15,14 @@
 from turing.router.config.resource_request import ResourceRequest
 from turing.router.config.log_config import LogConfig, ResultLoggerType
 from turing.router.config.router_config import RouterConfig, Protocol
-from turing.router.config.router_ensembler_config import EnsemblerStandardConfig, RouterEnsemblerConfig, \
-    StandardRouterEnsemblerConfig
+from turing.router.config.router_ensembler_config import StandardRouterEnsemblerConfig
```

Comment:
This file and the other SDK-based e2e tests have simply been re-formatted (and some unused imports removed).